test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
good clarification!

1. 
  data:
    - blog
2. 
  data: 
    - blog 
  included: 
    - posts
 - I guess this could be up for debate, but I think for deeply nested includes, it *should* rely on the the controller include option, to reduce accidental / unavoidable data inclusion. i.e.: I think default_includes on only the top level serializable object allows  for the most flexibility

3. 
  data: 
    - blog
  include: 
    - posts
    - category